tinyagentos/projects/doc_review_store.py

```
@@ -0,0 +1,149 @@
+        if new_state == "approved":
+            sets.append("reviewed_by = ?")
+            sets.append("reviewed_at = ?")
+            params.extend([actor_id, now])
+        elif new_state == "changes_requested":
+            sets.append("changes_requested_by = ?")
+            sets.append("changes_requested_at = ?")
+            params.extend([actor_id, now])
+
+        params.extend([project_id, doc_path])
+        await self._db.execute(
+            f"UPDATE doc_reviews SET {', '.join(sets)} WHERE project_id = ? AND doc_path = ?",
```

Ruff (0.16.0)

[error] Possible SQL injection vector through string-based query construction

(S608)